locales/en/messages.json

```diff
@@ -789,6 +789,10 @@
         "message": "Runway Takeoff Prevention has been triggered",
         "description": "Description of the RUNAWAY_TAKEOFF arming disable flag"
     },
+    "initialSetupArmingDisableFlagsTooltipCRASH": {
+        "message": "Ouch!",
```

Comment:
It's funny, but I don't know if this is the correct description :)

I usually took them from here: https://github.com/betaflight/betaflight/wiki/Arming-Sequence-&-Safety but it seems it has neither been added to the wiki.

Reply:
"Disarmed via crash detection"

"Apparently, we've crashed"

"Crash has been detected"

Its a fairly recent status flag, that is very unlikely to show on the configurator.

Reply:
If we set it to something funny now I am almost certain we will forget to change it back before the next release. "Disarmed via crash detection" sounds good.

Reply:
thanks for choosing. :)